osidb_bindings/bindings/python_client/models/affect_bulk_put_request.py

```diff
@@ -181,6 +183,8 @@ def from_dict(cls: type[T], src_dict: dict[str, Any]) -> T:
         def _parse_flaw(data: object) -> Union[None, UUID]:
             if data is None:
                 return data
+            if isinstance(data, Unset):
```

Comment:
Cannot it be simplified like this?
```py
             if data is None or isinstance(data, Unset):                                                                                                                                    
                  return data
 ```

Reply:
It is a generated code from the jinja templates (e.g. https://github.com/openapi-generators/openapi-python-client/blob/v0.22.0/openapi_python_client/templates/property_templates/union_property.py.jinja) and parts of the generated code are conditional and modular so yes, I could definitely simplify it but my aim was rather to make the template treat all the fields as non-required and not doing code optimizations of the generated code.

Reply:
I see.  It is better to make the template code easier to read rather than simplifying the generated code, which does not need to be maintained on its own.